### ISSUE TITLE
cmp_nvim_lsp.update_capabilities is deprecated

### ISSUE THREAD

**Issue by FoxySeta** [lucat1/visimp#43]
On visimp startup:
```
cmp_nvim_lsp.update_capabilities is deprecated, use cmp_nvim_lsp.default_capabilities instead. See :h deprecated
This function will be removed in cmp-nvim-lsp version 1.0.0
stack traceback:
        ...ite/pack/paks/opt/cmp-nvim-lsp/lua/cmp_nvim_lsp/init.lua:68: in function 'update_capabilities'
        ...im/site/pack/paks/start/visimp/lua/visimp/layers/cmp.lua:87: in function 'preload'
        ...e/nvim/site/pack/paks/start/visimp/lua/visimp/loader.lua:101: in function 'preload'
        ...re/nvim/site/pack/paks/start/visimp/lua/visimp/setup.lua:31: in function 'next'
        ...re/nvim/site/pack/paks/start/visimp/lua/visimp/setup.lua:103: in function <...re/nvim/site/pack/paks/start/visimp/lua/visimp/setup.lua:42>
        /home/stefano/.config/nvim/init.lua:1: in main chunk
```

**Comment by mustafakibar** [comment]
I'm using nvim v.0.7.2(latest) in my ubuntu and I got this error too.

I used the following `grep -Ril "cmp_nvim_lsp" ~/.config/nvim` command to find which files contain `cmp_nvim_lsp.update_capabilities` then changing the `cmp_nvim_lsp.update_capabilities` to the `use cmp_nvim_lsp.default_capabilities`. And the message is gone.

**Comment by lucat1** [comment]
I’d say your command does not fix this issue were it to happen with visimp. This error has already been patched within this project in #44 and a simple update would solve your issue. The command you gave is useful for a custom neovim config though!
